tests/utils.py

```diff
@@ -125,18 +135,20 @@ def put_bad_conda_on_path(testing_workdir):
     except:
         raise
```

Comment:
```raise``` could have a message which would be cleaner if the test case failed

Reply:
This re-raises the exception that triggered the except clause, with whatever message it already had